src/i18n/ja/nav.ts

```diff
@@ -6,14 +6,14 @@ export default NavDictionary({
 	'install-and-setup': 'インストールとセットアップ',
 	'guides/deploy': 'サイトのデプロイ',
 	'guides/migrate-to-astro': 'Astroへの移行',
-	'blog-tutorial': 'ブログを作る',
+	'blog-tutorial': 'チュートリアル： ブログを作る',
```

Comment:
全角の「：」を使用する場合、半角スペースは不要かなと思います:

```suggestion
	'blog-tutorial': 'チュートリアル：ブログを作る',
```

あるいは「:」の方に変えても良さそうです:

```suggestion
	'blog-tutorial': 'チュートリアル: ブログを作る',
```